test/WebJobs.Script.Tests.Integration/EndToEndTestsBase.cs

```diff
@@ -289,6 +289,39 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(TestHelpers.RemoveByteOrderMarkAndWhitespace(id), TestHelpers.RemoveByteOrderMarkAndWhitespace(result));
         }
 
+        protected async Task CosmosDBTriggerToBlobTestImpl()
+        {
+            // DocumentDB tests need the following environment vars:
+            // "AzureWebJobsDocumentDBConnectionString" -- the connection string to the account
+            await Fixture.InitializeDocumentClient();
+            bool collectionsCreated = await Fixture.CreateDocumentCollections();
+            var resultBlob = Fixture.TestOutputContainer.GetBlockBlobReference("completed");
```

Comment:
minor: just to avoid confusion -- can you name this "cosmosdbtriggere2e-completed"? I know we use "completed" for some others, but it's nicer if it's clear.